packages/devextreme/js/__internal/ui/list/m_list.edit.decorator.static.ts

```diff
@@ -1,25 +1,27 @@
+import type { dxElementWrapper } from '@js/core/renderer';
 import $ from '@js/core/renderer';
+import type { ClickEvent } from '@js/ui/button';
 import Button from '@js/ui/button';
 
+import type { BagConfig } from './m_list.edit.decorator';
 import EditDecorator from './m_list.edit.decorator';
 import { register as registerDecorator } from './m_list.edit.decorator_registry';
 
 const STATIC_DELETE_BUTTON_CONTAINER_CLASS = 'dx-list-static-delete-button-container';
 const STATIC_DELETE_BUTTON_CLASS = 'dx-list-static-delete-button';
 
 class EditDecoratorStatic extends EditDecorator {
-  afterBag(config?) {
-    const { $itemElement } = config;
-    const { $container } = config;
+  afterBag(config?: BagConfig): void {
+    const { $itemElement = $(), $container = $() } = config ?? {};
 
     const $button = $('<div>').addClass(STATIC_DELETE_BUTTON_CLASS);
 
     this._list._createComponent($button, Button, {
```

Comment:
The _createComponent function incorrectly handles types for Button and CheckBox because they don’t extend the base Component class and are renovated. We need to update its typings so that the third parameter can be ButtonOptions or CheckBoxOptions